src/api/server/common.rst

```diff
@@ -1210,9 +1215,11 @@ error.
     :>json string target: Replication target
     :>json string start_time: Timestamp of when the replication was started
     :>json string last_update: Timestamp of last state update
-    :>json object info: May contain additional information about the state.
-                        For error states, this will be a string. For success
-                        states this will contain a JSON object (see below).
+    :>json object info: May contain additional information about the
```

Comment:
Should this say "May" or "Will"? Is it ever blank?

Reply:
Agree